packages/core/src/lib/services/fit-bounds.ts

```diff
@@ -49,9 +49,13 @@ export class FitBoundsService {
   private _generateBounds(
     includeInBounds: Map<string, google.maps.LatLng | google.maps.LatLngLiteral>
   ) {
-    const bounds = new google.maps.LatLngBounds();
-    includeInBounds.forEach(b => bounds.extend(b));
-    return bounds;
+    if (includeInBounds.size === 0) {
+      return null;
+    } else {
```

Comment:
You don't need to put the remaining code in an else block because it can only be reached when the if condition is false.